packages/cwd-macros/src/lib.rs

```diff
@@ -109,6 +111,7 @@ fn cwd_interface_path(inside: &str) -> Path {
 /// ```
 #[proc_macro_attribute]
 pub fn voting_query(metadata: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
naming nit: `voting_module_query` to match `proposal_module_query`

Reply:
Nice, updated.